ext/msgpack/packer.h

```diff
@@ -31,15 +31,15 @@ typedef struct msgpack_packer_t msgpack_packer_t;
 struct msgpack_packer_t {
     msgpack_buffer_t buffer;
 
-    bool compatibility_mode;
-    bool has_bigint_ext_type;
-    bool has_symbol_ext_type;
-
     ID to_msgpack_method;
     VALUE to_msgpack_arg;
 
     VALUE buffer_ref;
 
+    bool compatibility_mode;
+    bool has_bigint_ext_type;
+    bool has_symbol_ext_type;
+
     /* options */
     bool comaptibility_mode;
```

Comment:
Btw, if you use a bitfield, you can even turn this from 16B down to 8B

Reply:
I'm never too sure how bitfields impact performance. Probably negligible, but I only made this change because it was an obvious small waste.